src/add_loads.jl

```diff
@@ -2,13 +2,13 @@ function maybeaddref!(ls::LoopSet, op)
     ref = op.ref
     id = findfirst(r -> r == ref, ls.refs_aliasing_syms)
     # try to CSE
-    if isnothing(id)
+    if id === nothing
```

Comment:
I should perhaps replace all `isnothing(x)` with `x === nothing`.

Reply:
Yeah I think that's a good idea.